tutorials-v5/time-evolution/013_nonmarkovian_monte_carlo.md

```diff
@@ -518,13 +521,225 @@ ax.set_ylim(-0.02, 0.22)
 plt.show()
 ```
 
+<!-- #region -->
+### Monte Carlo Simulations on Computing Clusters via MPI
+
+To improve the convergence behavior of the example above at long times, many trajectories are needed.
+We use this fact as a motivation to demonstrate QuTiP's MPI capabilities.
+On the QuTiP side, running Monte Carlo simulations on massively parallel infrastructure through MPI is as easy as replacing `'map': 'parallel'` by `'map': 'mpi'` in the provided options.
+QuTiP will then use the `MPIPoolExecutor` provided by the `mpi4py` module to simulate the trajectories in parallel.
+Such calculations can typically not be started from within a Jupyter notebook.
+The code below is an example for a standalone script that could be submitted to a job scheduler on a supercomputer.
+
```

Comment:
I like having the code and SLURM examples in the markdown.  Maybe this can also be added to the docs in some form, if not already?  Also, is it worth adding a similar example to one of the mcsolve notebooks? (perhaps not neccessary, or maybe later if we find an example that would benefit a lot from mpi)